crates/mun_codegen/src/ir.rs

```diff
@@ -192,13 +190,6 @@ impl<'ink, S: BasicType<'ink>, T: IsIrType<'ink, Type = S>> IsPointerType<'ink>
     }
 }
 
-// HACK: Manually add `*const TypeInfo`
-impl<'ink> IsPointerType<'ink> for *const TypeInfo {
-    fn ir_type(context: &'ink Context, _target: &TargetData) -> PointerType<'ink> {
-        context.i8_type().ptr_type(AddressSpace::Generic)
-    }
-}
-
 // HACK: Manually add `*const c_void`
```

Comment:
Why do we still need this one?

Reply:
We still need these because we cant add a type for `std::ffi::c_void` because it has an undefined type. But the places where we use it, we always use it as a pointer, therefor we only add the information for a pointer to `std::ffi:c_void`.

Reply:
In that case, is this a hack or just part of supporting our required intrinsics?

Or is it a hack because we don't deduce the intrinsics types? 🤔

Reply:
Yeah, you're right, Ill remove the comment